source/_config.yml

```diff
@@ -582,6 +590,19 @@ documents:
     version: "1"
     nav: /_includes/hiera1.html
 
+  /puppetserver/6.3:
+    doc: puppetserver
+    version: "6.3"
+    nav: /puppet/6.3/_puppet_toc.html
+    external_source:
+      repo: git://github.com/puppetlabs/puppetserver.git
+      commit: origin/master
+      subdirectory: documentation
+    my_versions:
+      puppet: "6.4"
+      facter: "3.13"
+    hide: true
+
```

Comment:
Below, in the puppetserver/6.2 section, it lists puppet "6.2" under my_versions. This version of Puppet Server is relevant to two versions of Puppet: 6.2 and 6.3. Do I need to list puppetserver again, or add additional information here to reflect that this version is ALSO in Puppet 6.3?